internal/venice-common/src/main/java/com/linkedin/venice/meta/VersionStatus.java

```diff
@@ -8,21 +8,51 @@
 
 
 /**
- * Enums of status of verion.
+ * Enums of status of version.
  */
 public enum VersionStatus implements VeniceEnumValue {
-  NOT_CREATED(0), STARTED(1),
-  // Version has been pushed to venice(offline job related to this version has been completed), but is not ready to
-  // serve read request because the writes to this store is disabled.
+  /**
+   * This version hasn't been created yet. It is not persisted in ZK
+   */
+  NOT_CREATED(0),
+
+  /**
+   * Version has been pushed to venice(offline job related to this version has been completed), but is not ready to
+   * serve read request because the writes to this store is disabled
+   */
+  STARTED(1),
+
+  /**
+   * Version has been pushed to venice and is ready to serve read request. Intermediate status after a push job succeeds before DeferredVersionSwapService
```

Comment:
Intermediate status after a push job finished pushing data to the child colo before DeferredVersionSwapService flips the status to ONLINE. This status only exists in the parent